pkgs/os-specific/linux/vendor-reset/default.nix

```diff
@@ -17,6 +17,14 @@ stdenv.mkDerivation rec {
     hash = "sha256-Klu2uysbF5tH7SqVl815DwR7W+Vx6PyVDDLwoMZiqBI=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "fix-linux-6.12-build.patch";
+      url = "https://github.com/gnif/vendor-reset/compare/084881c6e9e11bdadaf05798e669568848e698a3...54ffd6a012e7567b0288bc5fcc3678b545bd5aec.patch";
```

Comment:
GitHub will (eventually) garbage collect your commits if you force push your PR again, so this is not a stable URL.  For unmerged GitHub PRs, the patch contents should just be included in Nixpkgs.

Reply:
I did so and performed a force push (see 6f402f733481343c0bf4a0656a1110d027dfb5d4). Thanks for the feedback :)